core/src/scheme/scheme.rs

```diff
@@ -35,11 +35,6 @@ pub struct Scheme {
     pub name: String,
     /// What engine are we using for this?
     pub engine: Arc<dyn ConsensusEngine>,
-    /// Name of the subdir inside the main data dir to use for chain data and settings.
-    pub data_dir: String,
-
-    /// Known nodes on the network in enode format.
-    pub nodes: Vec<String>,
```

Comment:
How are these used in CodeChain?

Reply:
They were not used in CodeChain either. It seems that we copied scheme file Ethereum.

Reply:
Ok.